src/components/sectioningConfirm/confirmLabware.machine.ts

```diff
@@ -159,6 +177,12 @@ export const createConfirmLabwareMachine = (
             UPDATE_SECTION_NUMBER: {
               actions: ["updateSectionNumber", "commitConfirmation"],
             },
+            UPDATE_ALL_SECTION_NUMBERS: {
+              actions: ["updateAllSectionNumbers", "commitConfirmation"],
+            },
+            SET_MIN_SECTION_NUMBER: {
+              actions: ["setMinimumSectionNumber", "commmitConfirmation"],
```

Comment:
"commmitConfirmation" should presumably be "commitConfirmation"

Reply:
Typo- will correct.